src/services/user.service.js

```diff
@@ -0,0 +1,36 @@
+const { User } = require('../models/User.model');
+const { generateRandomId } = require('../Random');
+
+const getAll = async () => {
+  return User.findAll();
+};
+
+const getById = async (id) => {
+  return User.findByPk(+id);
+};
+
+const updateUser = async ({ id, name }) => {
+  await User.update({ name }, { where: { id } });
+};
+
+const createUser = async (name) => {
+  const id = generateRandomId();
+
+  return User.create({ id, name });
+};
+
+const deleteUser = async (id) => {
+  await User.destroy({
```

Comment:
In the 'deleteUser' function, there is no error handling or feedback to indicate whether the deletion was successful or not. Consider returning the result of the 'User.destroy' function to give feedback to the caller of this function.